lib/index.js

```diff
@@ -1 +1,3 @@
 require ('./css/styles');
+
+const firebase = require('./firebase');
```

Comment:
I _think_ you only need to require the file here.  It was my understanding that you only needed to declare a const if you're creating a function that uses it later in the file.  is there a way to test at this point or will we not know until we start building out the javascript later?